x-pack/test_utils/chrome_extension/test_subjects_finder/popup.js

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+/* eslint-disable no-undef */
+
+// const trace = (message) => {
```

Comment:
Do we want to keep this commented-out code or can it be removed?

Reply:
While developing the extension I removed it then had to re-write it multiple times. It is useful any time we need to output some debug information.